client/broadcaster/serial.go

```diff
@@ -27,7 +27,7 @@ var (
 
 	// errors are of the form:
 	// "account sequence mismatch, expected 25, got 27: incorrect account sequence"
-	recoverRegexp = regexp.MustCompile("^account sequence mismatch, expected (\\d+), got (\\d+):")
+	recoverRegexp = regexp.MustCompile(`^account sequence mismatch, expected (\\d+), got (\\d+):`)
```

Comment:
🚀

Reply:
not sure this makes a difference here :shrug: 

does `\` still need to be escaped?  I guess so if the tests are passing.

Reply:
It's been changed now. This change comes from the code linters requirements btw